CHANGES.rst

```diff
@@ -767,97 +767,12 @@ Other Changes and Additions
 
 - Removed dependency on scikit-image. [#10214]
 
-
-4.0.6 (unreleased)
```

Comment:
I didn't think we should have unreleased sections in the changelog now we are using towncrier, but can revert if I am missing something.